stable/dask/templates/_helpers.tpl

```diff
@@ -0,0 +1,31 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "dask.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
```

Comment:
Fullname templates should be based on the standard fullname template and append a component suffix.

Is there a reason why you make the components dynamic? I'd just hard-code them as you have them in `values.yaml`.